packages/common/src/hooks/purchaseContent/useChallengeCooldownSchedule.ts

```diff
@@ -11,6 +12,8 @@ import { COOLDOWN_DAYS } from './constants'
 
 const { getUndisbursedUserChallenges } = audioRewardsPageSelectors
 
+dayjs.extend(utc)
```

Comment:
This makes me slightly nervous, as we're changing the global configuration of dayjs from within an arbitrary module. I think it might make sense to create a module that configures dayjs _once_ and import that early in the application instead of having various configuration statements scattered around the codebase.
What do you think?

Reply:
good callout! ill add a ticket

Reply:
https://linear.app/audius/issue/PAY-2068/configure-dayjs-only-once